airflow/models/baseoperator.py

```diff
@@ -1064,10 +1069,23 @@ def _set_relatives(self,
                        task_or_task_list: Union['BaseOperator', List['BaseOperator']],
```

Comment:
I think we need to adjust this type annotation. But it will be hard due to circular imports.